imports/plugins/core/orders/server/no-meteor/schemas/schema.graphql

```diff
@@ -23,6 +23,17 @@ extend type Query {
 }
 
 extend type Mutation {
+  """
+  Use this mutation to cancel one item of an order, either for the full ordered quantity
+  or for a partial quantity. If partial, the item will be split into two items and the
+  original item will have a lower quantity and will be canceled.
```

Comment:
This is a good comment to use inline in the code where I asked for more commenting in the mutation. This makes sense, and having it inline there would be helpful.